src/main/java/org/gitlab4j/api/models/Topic.java

```diff
@@ -0,0 +1,71 @@
+package org.gitlab4j.api.models;
+
+import org.gitlab4j.api.utils.JacksonJson;
+
+public class Topic {
+
+    private Integer id;
+
+    private String name;
+
+    private String title;
+
+    private String description;
+
+    private int total_projects_count;
```

Comment:
Should be:
```suggestion
    private int totalProjectsCount;
```

the JSON serialization is doing the conversion to attribute name `"total_projects_count"` already.

Reply:
Solved with 1cb36853bd56316acf99b6b7fbf9f48e4c094a5a